_posts/2021-12-08-#23443.md

```diff
@@ -12,13 +12,55 @@ commit:
 
 _Notes and questions to follow soon!_
 
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+* Erlay is a new transaction relay protocol which is supposed to reduce the bandwidth nodes use to
+  announce transactions.
+
+  - We looked at Erlay at the review club [previously](https://bitcoincore.reviews/18261).
+
+  - Now, we will review a particular subset of Erlay commits to be merged first.
+
+  - For this PR, it might make sense to overview the [full PR again](https://github.com/bitcoin/bitcoin/pull/21515)
+    (note that this is a different PR number from what's above because that one was closed), and
+    the [updated BIP](https://github.com/naumenkogs/bips/blob/bip_0330_updates/bip-0330.mediawiki).
+    Don't spend too much time on it though, the overall understanding of the protocol is sufficient.
+
+* We will review the first batch of commits enabling Erlay. In this PR
+
+  - A node becomes able to negotiate the support for Erlay with another node by sending/receiving
+    a new p2p message.
+
+  - Once the handshake is done, the node also initiates the "reconciliation state", a variable to
+    keep track of ongoing reconciliations with a particular peer.
+
+  - If the peer is disconnected, the corresponding reconciliation state should be cleared.
+
+* We moved forward with this PR once [minisketch](https://bitcoinops.org/en/topics/minisketch/) was
+  merged, although minisketch is not required for this particular Erlay sub-PR. Another dependency
+  was [WTXID PR](https://bitcoincore.reviews/18044).
+
 ## Questions
+
 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
 -->
 
+2. What are the benefits to splitting PRs into smaller parts? Are there any drawbacks to this
+   approach? What's different about treating sub-PRs?
+
+3. When are the nodes supposed to announce Erlay support? What they should consider before doing
+   so and in which cases it's clearly meaningless?
+
+4. What is the overall handshake and "registration for reconciliation" protocol flow?
+
+5. Why Erlay protocol version feature is useful? Under which conditions it should be bumped?
+
+6. What is the reason for generating reconciliation salt (`m_local_salts`)? How is it generated
+   and why?
+
+7. Why does it make sense in some cases to disconnect a protocol violating peer rather than
+   just ignoring the message?
```

Comment:
I recommend prompting the attendees to look into the code. Giving them a specific number of things to search for also helps.
```suggestion
7. This PR adds 4 new conditions under which nodes will disconnect a peer. What are they?
When does it make sense to disconnect a peer rather than just ignore the message?
```